scanpy/plotting/_anndata.py

```diff
@@ -1623,6 +1627,17 @@ def matrixplot(adata, var_names, groupby=None, use_raw=None, log=False, num_cate
 
     mean_obs = obs_tidy.groupby(level=0).mean()
 
```

Comment:
To avoid confusion of what is row and what is column I can think of two solutions. One is to duplicate the code in each of the large swap blocks. The other is to add here something like:
```PYTHON
if swap_axes:
    standandar_scale = 'row' if standard_scale == 'col' else 'col'
```

Reply:
OK, I think second way is a bit cleaner. Let me know if it's ok now.

Reply:
Actually, what may be better is to set the options for standar_scale as `category` or `var` as this is always clear what to scale no matter the orientation. What do you think?

Reply:
This is a great idea! Now I changed it to var and group.